src/main/java/com/mariamkatamashvlii/gym/model/User.java

```diff
@@ -0,0 +1,50 @@
+package com.mariamkatamashvlii.gym.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "user")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "userId")
```

Comment:
One general comment , once name of table/field matche to correspondent name on database structure you can skip adding annotation Table and Column.
The idea is that: do not write the lines which can be omitted as being served by frameworks.